Formula/logcli.rb

```diff
@@ -1,9 +1,9 @@
 class Logcli < Formula
   desc "Run LogQL queries against a Loki server"
   homepage "https://grafana.com/loki"
-  url "https://github.com/grafana/loki/archive/v2.2.1.tar.gz"
-  sha256 "4801a9418c913bcca5e597d09f0f7ce1f5a7ce879f8dba3e8fe86057cb592bcf"
-  license "Apache-2.0"
+  url "https://github.com/grafana/loki/archive/v2.3.0.tar.gz"
+  sha256 "c71174a2fbb7b6183cb84fc3a5e328cb4276a495c7c0be8ec53c377ec0363489"
+  license "AGPL-3.0-only"
```

Comment:
audit failed, chose the `-only` license to match grafana: https://github.com/Homebrew/homebrew-core/blob/c7f20f53695ff913ebacd06e824d76b699724874/Formula/grafana.rb#L6

Reply:
> Grafana Loki is distributed under AGPL-3.0-only. For Apache-2.0 exceptions, see LICENSING.md.

https://github.com/grafana/loki#license